src/libtrx/include/libtrx/game/level/common.h

```diff
@@ -2,4 +2,8 @@
 
 #include "../../virtual_file.h"
 
+#if TR_VERSION == 1
 void Level_ReadRoomMesh(int32_t room_num, VFILE *file);
+void Level_ReadObjectMeshes(
```

Comment:
Could we consider keeping this API available but unused, or possibly move it back to TR1? So far, we've only been diverging types and embedded data, and this would set a new precedent by starting to conditionalize the APIs we offer in libtrx. I'd avoid that if we can.

Reply:
Updated now so the API is similar for 1 and 2. The only difference is TR2 has no concept of `ROOM_MESH` yet because the parsing is linked to the property on the room, which hasn't been changed over yet (so same place as we were in #1938).
We'll just need to be aware not to use the other functions yet in TR2, which I think is OK for now.